spec/reek/cli/json_report_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'reek/examiner'
+require 'reek/cli/report/report'
+require 'reek/cli/report/formatter'
+
+include Reek
+include Reek::Cli
```

Comment:
I know we are doing this a lot in our codebase but at least for new specs, lets not do the "include Reek" dance and use fully qualified constants.